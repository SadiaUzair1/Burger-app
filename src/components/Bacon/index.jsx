export const Bacon = () => (
  <div>
    <img src={'/images/bacon.png'} width={550} height={40} alt={''} />
  </div>
)
