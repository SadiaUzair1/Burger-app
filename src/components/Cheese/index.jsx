export const Cheese = () => (
  <div>
    <img src={'/images/cheese.png'} width={700} height={25} alt={''} />
  </div>
)
