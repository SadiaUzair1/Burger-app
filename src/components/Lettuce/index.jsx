export const Lettuce = () => (
  <div>
    <img src={'/images/lettuce.png'} width={600} height={50} margin={50} alt={''} />
  </div>
)
