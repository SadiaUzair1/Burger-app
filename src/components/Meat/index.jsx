export const Meat = () => (
  <div>
    <img src={'/images/meat.png'} width={500} height={30} alt={''} />
  </div>
)
