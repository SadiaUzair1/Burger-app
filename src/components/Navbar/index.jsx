export const NavBar = () => (
  <nav className={'navbar navbar-light'}>
    <div className={'container-fluid'}>
      <a className={'navbar-brand'} href={'#'}>
        <img
          src={'/images/burger.png'}
          alt={''}
          width={'50'}
          height={'40'}
          className={'align-text-top m-1'}
        />
      </a>
    </div>
  </nav>
)
