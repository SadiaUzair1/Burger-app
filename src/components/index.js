export { Bacon } from 'components/Bacon'
export { Cheese } from 'components/Cheese'
export { Lettuce } from 'components/Lettuce'
export { Meat } from 'components/Meat'
export { NavBar } from 'components/Navbar'
