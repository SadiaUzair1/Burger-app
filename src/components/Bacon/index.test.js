import { Bacon } from 'components/Bacon'
import { mount, shallow } from 'enzyme'

import { findByTestAttr } from '../../../utlis'

const setup = () => {
  const wrapper = shallow(<Bacon />)
  return wrapper
}

const mounting = () => {
  const wrapper1 = mount(<Bacon />)
  return wrapper1
}

describe('rendering components', () => {
  let component
  let wrapper

  beforeEach(() => {
    component = setup()
    wrapper = mounting()
  })

  it('bacon should render alone', () => {
    expect(component.contains(<Bacon />))
  })

  it('bacon is rendering with children', () => {
    expect(wrapper.contains(<Bacon />))
  })

  it('image should be of becon', () => {
    const logo = findByTestAttr(component, 'bacon')
    expect(logo.length).toBe(1)
  })

  it('bacon should be rendered', () => {
    expect(component).toMatchSnapshot()
  })
})
