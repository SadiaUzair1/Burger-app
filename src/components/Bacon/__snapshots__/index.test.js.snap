// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering components bacon should be rendered 1`] = `
<div>
  <img
    alt=""
    data-test="bacon"
    height={40}
    src="/images/bacon.png"
    width={550}
  />
</div>
`;
